revscoring/utilities/train_test.py

```diff
@@ -127,8 +127,10 @@ def _train_test(scorer_model, observations, test_prop):
 
     logger.info("Training model...")
     scorer_model.train(train_set)
+    scorer_model.train_set = train_set
```

Comment:
Thanks for the input, I thought about it too, and it's my great concern right now. Do you think other attributes of scorer_model such as "stats" or "test_statistics" are enough or we should attach all values of the test to it? Also, we don't need to store the train set at all probably.
My reason to store is to make it pickle-able so we can load it later and use it in the stand-alone script for example. If you think another approach is better, please do tell and I'll implement it.